libraries/base/Data/Semigroup.hs

```diff
@@ -483,8 +496,6 @@ instance Applicative Option where
 
   Option Nothing  *>  _ = Option Nothing
   _               *>  b = b
-
--- | @since 4.9.0.0
```

Comment:
Why was this removed? `@since` tags are fantastic for figuring out version bounds at a glance.

Reply:
thanks, that was non-intentional.